src/phylum/ci/cli.py

```diff
@@ -61,6 +62,63 @@ def detect_ci_platform(args: argparse.Namespace, remainder: List[str]) -> CIBase
     return ci_env
 
 
+def ensure_project(ci_env: CIBase) -> None:
+    """Ensure a Phylum project is created and in place.
+
+    When a project is specified through arguments, attempt to create that project,
+    overwriting any `.phylum_project` file that already exists.
+    Continue on without error when a specified project already exists.
+    When a group is specified through arguments, attempt to use that group.
+    """
+    if not ci_env.phylum_project:
+        return
+
+    cmd = f"{ci_env.cli_path} project create {ci_env.phylum_project}"
+    if ci_env.phylum_group:
+        ensure_group(ci_env)
```

Comment:
Automatically creating groups is entirely different than automatically creating projects.

Assuming best practices, CI will be running as some sort of bot user. But we almost certainly do not want the bot user to be the owner of the group. And configuring access to the group is outside of the scope of this tool.

```suggestion
```